__main__.py

```diff
@@ -1,10 +1,13 @@
 from src.SpellingBee import CLI
 from src.SpellingBee import CommandHandler
+from src.SpellingBee import MakePuzzle
 import os
 
 CLI.drawTextBox(['Welcome to Spelling Bee! \ Presented by G(J)IFY',
-    'To start a new game, type \"!new\". \ To load a previous save, type \"!load\"',
-    'For a list of available commands, type \"!help\"'], 40, '^')
+    'To start a new game, type \"!new\". \ To load a previous save, type \"!load\"'], 40, '^')
+
+# TODO, restrict input to JUST !new, !load, and !exit
```

Comment:
Comment from Meeting: This will handle creating puzzle for either a new game OR a reload game. Implementation to come.